models/ecoli/analysis/variant/ribosome_crowding.py

```diff
@@ -0,0 +1,168 @@
+"""
+Produces a txt file for all, early, and late generations containing a list of
+genes and the index of variants where those genes were overcrowded by
+ribosomes. Here, overcrowded by ribosomes is defined as the actual
+probability of translation per transcript being less than the target
+probability of translation per transcript on average for at least one
+generation in at least one seed for that variant index.
+"""
+
+import numpy as np
+import pickle
+import os
+from wholecell.io.tablereader import TableReader
+from models.ecoli.analysis import variantAnalysisPlot
+from wholecell.analysis.analysis_tools import read_stacked_columns, \
+	stacked_cell_threshold_mask
+
+# 1 to exclude cells that took full MAX_CELL_LENGTH, 0 otherwise
+exclude_timeout_cells = 1
+
+"""
+1 to plot early (before MIN_LATE_CELL_INDEX), and late generations in
+addition to all generations
+"""
+exclude_early_gens = 1
+
+FONT_SIZE=9
+MAX_VARIANT = 43 # do not include any variant >= this index
+MAX_CELL_INDEX = 16 # do not include any generation >= this index
+
+"""
+generations before this may not be representative of dynamics 
+due to how they are initialized
+"""
+MIN_LATE_CELL_INDEX = 4
+
+MAX_CELL_LENGTH = 36000
```

Comment:
Might be good to specify what units this is as a comment - I'm assuming this is in seconds?